news/2021-01-14-performance-points-updates.md

```diff
@@ -0,0 +1,108 @@
+
+Thanks to the effort of a number of public contributors, we're excited to announce a new set of changes to osu!'s performance points (pp) algorithm that should help freshen things up at high levels of play. Read on for more details!
+
+![](/wiki/shared/news/2021-01-14-performance-points-updates/star-rating.jpg)
+
+Set off during the last few days of 2020, these new incoming changes represent an effort from various members of the community to try and address long-standing issues with one of the game's most important progression systems.
+
+For those unfamiliar with the technical minutae of the system, please consult [the wiki article](/wiki/Performance_points/) for an overview of the general ideas involved. 
+
+To summarize the article very briefly (you should really read the whole article), pp encompasses several core values as follows:
+
+- **Aim**, which considers how difficult it is to consistently hit notes back-to-back during play (affected by things like approach rate, jumps, Flashlight, Hidden, Hard Rock and more)
+- **Speed**, which considers how quickly new elements are presented to the player during the course of the map (affected by things like the number of hit objects over time, BPM, Double Time and Half Time)
+- **Accuracy**, which considers the player's ability to stay on time rhythmically and how difficult this is to achieve (affected by things like Hidden, Hard Rock, Flashlight and more)
+- **Strain**, which considers how often and how long a player has to endure intense or technically demanding patterns during play (affected by things like streams, fast jumps and more)
+
+**The changes listed below apply solely to the osu! game mode.**
+
+Let's get started with the details:
+
+## Approach Rate 11 Changes 
+
+Previously, any map with an approach rate (hereafter referred to as AR) higher than 10.33 was affected by a buff that increased the value of the speed and aim components by a flat 20%.
+
+Maps at or above AR 11 are often considered to be similar to the Flashlight mod in that they become considerably more difficult as a map progresses. This fact was not reflected in the previous version of the algorithm with its static 20% increase.
+
+As suggested by [**Xexxar**](https://osu.ppy.sh/users/2773526), the following change has been applied:
+
+- The buff for AR 11 maps now scales from +0% at 0 objects, to 26.66% at 1000 objects. The previous 20% buff is now attained at 750 objects instead.
+    + Old value: 20% flat.
+
+In essence, this nerfs the amount of pp awarded for shorter AR 11 maps, but buffs the amount gained for longer, more difficult performances.
+
+## NoFail Changes
+
+Before today's changes, the performance points algorithm applied a flat 10% reduction in the maximum possible pp (not score) you could receive for a play made with the NoFail mod active.
+
+As suggested by GitHub contributor stanriders in [this pull request](https://github.com/ppy/osu-performance/pull/127), the following change has been applied:
```

Comment:
link to his osu profile (since you did that for xexxar)? https://osu.ppy.sh/users/7217455

Reply:
Might as well use `StanR` instead of `stanriders`

Reply:
applied on both counts